mm2src/adex_cli/src/helpers.rs

```diff
@@ -22,6 +23,15 @@ where
     writer
         .write(&data)
         .map_err(|error| error_anyhow!("Failed to write data into {file}: {error}"))?;
+
+    Ok(())
+}
+
+#[cfg(unix)]
+pub(crate) fn set_file_permissions(file: &str, unix_mode: u32) -> Result<()> {
```

Comment:
duplicate feedback from above: 

_is this advised? Wouldn't it be better/safer to assume config file has correct permission setting as opposed to implementing a de-facto "chmod" into mm2?
Rly concerned over likely unneeded filestream ops and speaking general - against this._